RFS/src/main/java/com/rfs/common/OpenSearchClient.java

```diff
@@ -64,6 +66,168 @@ public OpenSearchClient(ConnectionContext connectionContext) {
         this.failedRequestsLogger = failedRequestsLogger;
     }
 
+    public Version getClusterVersion() {
```

Comment:
The Metadata EndToEndTest should have triggered many of the codepathes in this file, maybe it wasn't run during CI. Need to investigate.

Reply:
Updated the build.gradle so jacocoTestReport has a dependency on slowTest